### PR TITLE
Add more watchpoint plots and source files

### PR DESCRIPTION
**Pull request by MakisH**
To be cleaned up according to what we decide.
Source data might or might not be included in the end.
The solid-openfoam mesh convergence study will be removed, see #515.

### CODE REVIEW

File: perpendicular-flap/README.md [hunk 1]
```diff
@@ -73,17 +73,37 @@ As we defined a watchpoint on the 'Solid' participant at the flap tip (see `prec
...
 * The CalculiX adapter only supports linear finite elements (deal.II uses 4th order, FEniCS 2nd order).
 * SU2 models a compressible fluid, OpenFOAM and Nutils an incompressible one.  
 
+### Looking closer
+
+Excluding the `solid-openfoam` (outlier, provided mainly for technical testing), let's look at an overview of different combinations.
+
+Comparison of the different flow solvers (incompressible `fluid-openfoam` and `fluid-nutils`, compressible `fluid-su2`, dummy `fluid-fake`):
+
+![Flap watchpoints using fluid-openfoam](images/tutorials-perpendicular-flap-displacement-flow-comparison-watchpoints.png)
```
Comment (uekerman) [inline perpendicular-flap/README.md:93]:
```suggestion
![Flap watchpoints using solid-calculix](images/tutorials-perpendicular-flap-displacement-flow-comparison-watchpoints.png)
```

File: perpendicular-flap/README.md [hunk 2]
```diff
@@ -73,17 +73,37 @@ As we defined a watchpoint on the 'Solid' participant at the flap tip (see `prec
...
+
+Combinations using the incompressible `fluid-openfoam` case:
+
+![Flap watchpoints using fluid-su2](images/tutorials-perpendicular-flap-displacement-openfoam-watchpoints.png)
```
Comment (uekerman) [inline perpendicular-flap/README.md:97]:
```suggestion
![Flap watchpoints using fluid-openfoam](images/tutorials-perpendicular-flap-displacement-openfoam-watchpoints.png)
```

File: perpendicular-flap/images/tutorials-perpendicular-flap-displacement-openfoam-convergence-watchpoints.png
Comment (uekerman) [inline perpendicular-flap/images/tutorials-perpendicular-flap-displacement-openfoam-convergence-watchpoints.png:1]: This one, you don't mention in the README. Still needed?

File: perpendicular-flap/README.md [hunk 3]
```diff
@@ -73,17 +73,37 @@ As we defined a watchpoint on the 'Solid' participant at the flap tip (see `prec
...
+
+Combinations (excerpt) using the incompressible `fluid-nutils` case:
+
+![Flap watchpoints using fluid-nutils](images/tutorials-perpendicular-flap-displacement-nutils-watchpoints.png)
```
Comment (uekerman) [inline perpendicular-flap/README.md:101]: Wouldn't it make more sense to show the compressible SU2 here? As incompressible solver, we have OpenFOAM above already.

File: perpendicular-flap/images/tutorials-perpendicular-flap-displacement-all-watchpoints.png
Comment (uekerman) [inline perpendicular-flap/images/tutorials-perpendicular-flap-displacement-all-watchpoints.png:1]: not rendered right now. keep?

File: perpendicular-flap/plog-solid-openfoam-displacement.sh
Comment (uekerman) [inline perpendicular-flap/plog-solid-openfoam-displacement.sh:1]: typo "plog"

**Review by uekerman — APPROVED** [review]
Nice job. Small tweaks only.

File: perpendicular-flap/images/tutorials-perpendicular-flap-displacement-openfoam-convergence-watchpoints.png
Comment (MakisH) [reply perpendicular-flap/images/tutorials-perpendicular-flap-displacement-openfoam-convergence-watchpoints.png:1]: I want to remove it before merging. It is not meant to be any kind of rudimentary study. I just wanted to communicate my observations with you and @philipcardiff.

File: perpendicular-flap/README.md (hunk 3 above)
Comment (MakisH) [reply perpendicular-flap/README.md:101]: I will add both, just forgot. Thanks for the observation!

File: perpendicular-flap/images/tutorials-perpendicular-flap-displacement-all-watchpoints.png
Comment (MakisH) [reply perpendicular-flap/images/tutorials-perpendicular-flap-displacement-all-watchpoints.png:1]: Would it be helpful anywhere? I am happy to delete it.

File: perpendicular-flap/plog-solid-openfoam-displacement.sh
Comment (MakisH) [reply perpendicular-flap/plog-solid-openfoam-displacement.sh:1]: Thanks! Will be removed anyway.

File: perpendicular-flap/images/tutorials-perpendicular-flap-displacement-all-watchpoints.png
Comment (uekerman) [reply perpendicular-flap/images/tutorials-perpendicular-flap-displacement-all-watchpoints.png:1]: let's delete

**Comment by MakisH** [comment]
This is now ready content-wise, but I would still like to replace the raw log files with a `.tar.gz` pushed to the LFS server. But I first need to figure out some infrastructure restrictions (@valentin-seitz help, please).

**Comment by MakisH** [comment]
I pushed a `watchpoints.tar.gz` to the LFS (I will add documentation) and tested with a fresh clone. This is ready to go.

Edit: The LFS URL does not work for me right now (Feb 2026). Here is the file from my local clone:

[watchpoints.tar.gz](https://github.com/user-attachments/files/25499893/watchpoints.tar.gz)

**Pull request merged**